contracts/src/NativeTokenDestination.sol

```diff
@@ -122,26 +122,17 @@ contract NativeTokenDestination is
      * @param settings Constructor settings for this destination token bridge instance.
      * @param nativeAssetSymbol The symbol of the native asset.
      * @param initialReserveImbalance The initial reserve imbalance that must be collateralized before minting.
-     * @param decimalsShift The number of decimal places to shift the token amount by.
-     * @param multiplyOnDestination See {TeleporterTokenDestination-multiplyOnDestination}.
      * @param burnedFeesReportingRewardPercentage_ The percentage of burned transaction fees
      * that will be rewarded to sender of the report.
      */
     constructor(
         TeleporterTokenDestinationSettings memory settings,
         string memory nativeAssetSymbol,
         uint256 initialReserveImbalance,
-        uint8 decimalsShift,
-        bool multiplyOnDestination,
         uint256 burnedFeesReportingRewardPercentage_
     )
         ERC20(string.concat("Wrapped ", nativeAssetSymbol), nativeAssetSymbol)
-        TeleporterTokenDestination(
-            settings,
-            initialReserveImbalance,
-            decimalsShift,
-            multiplyOnDestination
-        )
+        TeleporterTokenDestination(settings, initialReserveImbalance)
```

Comment:
Should we add a require statement that checks `settings.tokenDecimal` is 18 in this constructor?

Reply:
I think yes because we're directly instantiating `ERC20`, and it has the default decimals of `18`. Done in https://github.com/ava-labs/teleporter-token-bridge/pull/142/commits/a28972c7ed9415cec9cb714da02c94e175aca929